ui/pages/confirmations/hooks/useCurrentConfirmation.ts

```diff
@@ -76,6 +76,8 @@ const useCurrentConfirmation = () => {
         return;
       }
 
+      // comment if condition below to enable re-design for SIWE signatures
+      // this can be removed once SIWE code changes are completed
```

Comment:
I'm confused as to what the purpose of this comment is. could you elaborate/re-elaborate please?

Reply:
This is added to help in further work on SIWE signature. Commenting out this if condition will result in SIWE signatures shown in new designs.

Reply:
👍🏼 
```suggestion
      // comment out if block below to enable redesigned SIWE signatures
      // TODO: remove once redesigned SIWE code changes are completed
```